Apps/WebClient/src/ClientApp/src/components/navmenu/SidebarComponent.vue

```diff
@@ -135,14 +135,22 @@ export default class SidebarComponent extends Vue {
         this.setUserPreference({ preference });
     }
 
+    private get isQueuePage(): boolean {
```

Comment:
Should this be refactored into a shared code as it repeats in HeaderComonent as well?

Reply:
We can discuss with @MikeLyttle later.